src/librustc/diagnostics.rs

```diff
@@ -406,7 +407,7 @@ fn baz<'a>(x: &'a str, y: &str) -> &str { }
 Lifetime elision in implementation headers was part of the lifetime elision
 RFC. It is, however, [currently unimplemented][iss15872].
 
```

Comment:
The issue  #15872 is closed so this statement is no longer true. What should we do with this error code?

Reply:
IMHO, this whole paragraph should just be removed.

Reply:
I will open another PR to fix that after this one is merged.

Reply:
I opened a PR #58456